onnxruntime/core/graph/graph_utils.cc

```diff
@@ -448,6 +448,22 @@ size_t RemoveNodeOutputEdges(Graph& graph, Node& node) {
   return output_edges.size();
 }
 
+ONNX_NAMESPACE::ModelProto GetModelProto(const onnxruntime::GraphViewer& graph) {
+  ONNX_NAMESPACE::ModelProto model_proto;
+  *model_proto.mutable_graph() = graph.ToGraphProto();
+  model_proto.set_ir_version(ONNX_NAMESPACE::Version::IR_VERSION);
+  return model_proto;
```

Comment:
I think there're still other fields in ModelProto need to be set, for example, opset_import.

Reply:
Thanks! I have added it :)